.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: Build
+
+on:
+  push:
+    branches: master
+  pull_request:
+
+jobs:
+  build:
+    name: Test on Python ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10", "3.11", "3.12"]
```

Comment:
How did you select this? Is there some default set we can use so we don't need to update this in the future?